crates/transaction-pool/src/maintain.rs

```diff
@@ -535,6 +576,103 @@ fn changed_accounts_iter(
         .map(|(address, acc)| ChangedAccount { address, nonce: acc.nonce, balance: acc.balance })
 }
 
+async fn get_local_txs_backup<P>(pool: P, local_txs_backup: Option<PathBuf>)
+where
+    P: TransactionPoolExt + 'static,
+{
+    if let Some(file_path) = local_txs_backup.clone() {
+        trace!(target: "reth::cli", txs_file =?file_path, "Check local persistent storage for saved transactions");
+        match std::fs::read(&file_path) {
+            Ok(data) => {
+                let mut data_slice = data.as_slice();
+                while !data_slice.is_empty() {
```

Comment:
I think this can be decoded into a `Vec<Tx>` directly, and we then call add_transactions